integrations/cohere/src/haystack_integrations/components/rankers/cohere/ranker.py

```diff
@@ -0,0 +1,168 @@
+from typing import Any, Dict, List, Optional
+
+from haystack import Document, component, default_from_dict, default_to_dict, logging
+from haystack.lazy_imports import LazyImport
+from haystack.utils import Secret, deserialize_secrets_inplace
+
+logger = logging.getLogger(__name__)
+
+with LazyImport(message="Run 'pip install cohere'") as cohere_import:
+    import cohere
+logger = logging.getLogger(__name__)
```

Comment:
```suggestion
```
This can be removed.

This integration explicitly depends on Cohere client 
https://github.com/deepset-ai/haystack-core-integrations/blob/0fd7f9189eda5e3cd41e6705298816f2815b7ad3/integrations/cohere/pyproject.toml#L26

Reply:
Makes sense. Will make this change